Makefile

```diff
@@ -64,6 +68,13 @@ logs: ## display grafana logs (follow mode)
 	@$(COMPOSE) logs -f grafana
 .PHONY: logs
 
+plugins: ## download plugins
+	mkdir -p ./var/lib/grafana/plugins
+	curl -Lo /tmp/grafana_bar_chart_plugin.zip $(BAR_CHART_PLUGIN)
+	rm -rf ./var/lib/grafana/plugins/*
+	unzip /tmp/grafana_bar_chart_plugin.zip -d ./var/lib/grafana/plugins/
```

Comment:
We can make a rule per plugin and make the `plugins` rule a meta phony rule that installs them all. WDYT?

Reply:
Good Idea) Thanks)  Updated)